go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	golang.org/x/crypto v0.32.0
 	golang.org/x/mobile v0.0.0-20240716161057-1ad2df20a8b6
 	golang.org/x/net v0.34.0
+	golang.org/x/time v0.5.0
```

Comment:
why v0.5.0? v0.10.0 is available also (even v0.11.0 is, but that seems to require Go 1.23).

Reply:
Yeah I'm not sure, I think it was auto-added. Will change it

Reply:
Now that we are back on 1.23 it could also be 0.11.0 :nerd_face: not very important though I assume